apache/superset
# feat: custom content in alert and reports

### SUMMARY

Add text editor in alert/report modal to add custom html content for email and markdown content for slack reports.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before
![image](https://user-images.githubusercontent.com/12967587/152907861-e3dd3f6d-f95b-4fd6-a202-68b59c5b0c89.png)

After
![image](https://user-images.githubusercontent.com/12967587/152908162-44e56ae9-112e-4c74-857b-fb88d20cf2a7.png)



### New Dependencies
New dependency on `https://github.com/jpuri/react-draft-wysiwyg` for editor

### TESTING INSTRUCTIONS

Enable ALERTS_REPORTS feature flag
Set report for any chart/dashboard
Check received report -> should have custom body set in above step.

### TODO
- [ ] Update all other delivery methods to use custom content.
- [ ] Migration to add new column in table with `<a href="{chart_explore_url}">Explore in Superset</a>` link for existing reports.
- [ ] Update existing tests and add new for this feature.

### ADDITIONAL INFORMATION
